### PR TITLE
feat(docs): Update Install docs

### PR DESCRIPTION
**Pull request by simon-cj**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @gaocegege @FogDong 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: scripts/installation/install.sh [hunk 1]
```diff
@@ -20,21 +20,17 @@ export KLEVER_MODEL_REGISTRY_PORT=30100
 # 
 # Go to manifests directory, it is workdir.
 CRTDIR=$(pwd)
```
Comment (gaocegege) [inline scripts/installation/install.sh:22]: What's the meaning of CRTDIR?

Comment (simon-cj) [reply scripts/installation/install.sh:22]: CURRENT DIRECTORY

Comment (Thrimbda) [reply scripts/installation/install.sh:22]: how about just CWD? It's a convention

Comment (simon-cj) [reply scripts/installation/install.sh]: SGTM, addressed, PTAL

**Review by gaocegege — COMMENTED** [review]
/lgtm

**Comment by simon-cj** [comment]
/cc @Thrimbda 
The install script is ready, you can install by install.sh, have any question please call me by lark.

**Pull request merged**